config/application.rb

```diff
@@ -32,6 +32,7 @@ class Application < Rails::Application
     # Override rails template engine: erb to haml
     config.generators do |g|
       g.template_engine :haml
+      g.orm :active_record
```

Comment:
@Novsochetra what is this?

Reply:
@sophalsun I've copy it from buna. because I've faced some error, So should I've remove it ?